mediaplatform_jwp/models.py

```diff
@@ -273,6 +273,15 @@ def get_sources(self):
     #: A property which calls get_sources and caches the result.
     sources = cached_property(get_sources, name='sources')
 
+    def fetch_size(self):
+        """
+        Fetches the storage size of the video in bytes (the sum of all the sources).
+        Returns 0 if unable to get a size.
+
+        """
+        cache_resource = CachedResource.videos.filter(pk=self.pk).first()
```

Comment:
I'm wondering if we want to start adding ``mediaplatform_jwp.models.Video`` and ``...Channel`` foreign keys to ``CachedResource``. (Not for this story.) Then we can stop having to roll our own queries like this.